<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-0.9.305">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Vincent van Hoef">

<title>Analysis of human myeloid cell activation using ATAC sequencing</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
span.underline{text-decoration: underline;}
div.column{display: inline-block; vertical-align: top; width: 50%;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<script src="index_files/libs/clipboard/clipboard.min.js"></script>
<script src="index_files/libs/quarto-html/quarto.js"></script>
<script src="index_files/libs/quarto-html/popper.min.js"></script>
<script src="index_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="index_files/libs/quarto-html/anchor.min.js"></script>
<link href="index_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link id="quarto-text-highlighting-styles" href="index_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet">
<script src="index_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="index_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="index_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet">
<script async="" src="https://hypothes.is/embed.js"></script>
<script type="module" src="index_files/libs/quarto-ojs/esbuild-bundle.js"></script>
<link href="index_files/libs/quarto-ojs/quarto-ojs.css" rel="stylesheet">
<script type="ojs-define">
{"contents":[{"name":"insert_hist","value":["C0_B_R1","C0_B_R2","C0_R1","C0_R2","C1_R1","C1_R2","C24_R1","C24_R2","C4_R1","C4_R2","IK0_B_R1","IK0_B_R2","IK0_R1","IK0_R2","IK1_R1","IK1_R2","IK24_R1","IK24_R2","IK4_R1","IK4_R2"]}]}
</script>
<script type="ojs-define">
{"contents":[{"name":"contrast","value":["C0_B_R1_vs_C0_B_R2","C0_B_R1_vs_C0_R1","C0_B_R1_vs_C0_R2","C0_B_R1_vs_C1_R1","C0_B_R1_vs_C1_R2","C0_B_R1_vs_C24_R1","C0_B_R1_vs_C24_R2","C0_B_R1_vs_C4_R1","C0_B_R1_vs_C4_R2","C0_B_R1_vs_IK0_B_R1","C0_B_R1_vs_IK0_B_R2","C0_B_R1_vs_IK0_R1","C0_B_R1_vs_IK0_R2","C0_B_R1_vs_IK1_R1","C0_B_R1_vs_IK1_R2","C0_B_R1_vs_IK24_R1","C0_B_R1_vs_IK24_R2","C0_B_R1_vs_IK4_R1","C0_B_R1_vs_IK4_R2","C0_B_R2_vs_C0_R1","C0_B_R2_vs_C0_R2","C0_B_R2_vs_C1_R1","C0_B_R2_vs_C1_R2","C0_B_R2_vs_C24_R1","C0_B_R2_vs_C24_R2","C0_B_R2_vs_C4_R1","C0_B_R2_vs_C4_R2","C0_B_R2_vs_IK0_B_R1","C0_B_R2_vs_IK0_B_R2","C0_B_R2_vs_IK0_R1","C0_B_R2_vs_IK0_R2","C0_B_R2_vs_IK1_R1","C0_B_R2_vs_IK1_R2","C0_B_R2_vs_IK24_R1","C0_B_R2_vs_IK24_R2","C0_B_R2_vs_IK4_R1","C0_B_R2_vs_IK4_R2","C0_R1_vs_C0_R2","C0_R1_vs_C1_R1","C0_R1_vs_C1_R2","C0_R1_vs_C24_R1","C0_R1_vs_C24_R2","C0_R1_vs_C4_R1","C0_R1_vs_C4_R2","C0_R1_vs_IK0_B_R1","C0_R1_vs_IK0_B_R2","C0_R1_vs_IK0_R1","C0_R1_vs_IK0_R2","C0_R1_vs_IK1_R1","C0_R1_vs_IK1_R2","C0_R1_vs_IK24_R1","C0_R1_vs_IK24_R2","C0_R1_vs_IK4_R1","C0_R1_vs_IK4_R2","C0_R2_vs_C1_R1","C0_R2_vs_C1_R2","C0_R2_vs_C24_R1","C0_R2_vs_C24_R2","C0_R2_vs_C4_R1","C0_R2_vs_C4_R2","C0_R2_vs_IK0_B_R1","C0_R2_vs_IK0_B_R2","C0_R2_vs_IK0_R1","C0_R2_vs_IK0_R2","C0_R2_vs_IK1_R1","C0_R2_vs_IK1_R2","C0_R2_vs_IK24_R1","C0_R2_vs_IK24_R2","C0_R2_vs_IK4_R1","C0_R2_vs_IK4_R2","C1_R1_vs_C1_R2","C1_R1_vs_C24_R1","C1_R1_vs_C24_R2","C1_R1_vs_C4_R1","C1_R1_vs_C4_R2","C1_R1_vs_IK0_B_R1","C1_R1_vs_IK0_B_R2","C1_R1_vs_IK0_R1","C1_R1_vs_IK0_R2","C1_R1_vs_IK1_R1","C1_R1_vs_IK1_R2","C1_R1_vs_IK24_R1","C1_R1_vs_IK24_R2","C1_R1_vs_IK4_R1","C1_R1_vs_IK4_R2","C1_R2_vs_C24_R1","C1_R2_vs_C24_R2","C1_R2_vs_C4_R1","C1_R2_vs_C4_R2","C1_R2_vs_IK0_B_R1","C1_R2_vs_IK0_B_R2","C1_R2_vs_IK0_R1","C1_R2_vs_IK0_R2","C1_R2_vs_IK1_R1","C1_R2_vs_IK1_R2","C1_R2_vs_IK24_R1","C1_R2_vs_IK24_R2","C1_R2_vs_IK4_R1","C1_R2_vs_IK4_R2","C24_R1_vs_C24_R2","C24_R1_vs_C4_R1","C24_R1_vs_C4_R2","C24_R1_vs_IK0_B_R1","C24_R1_vs_IK0_B_R2","C24_R1_vs_IK0_R1","C24_R1_vs_IK0_R2","C24_R1_vs_IK1_R1","C24_R1_vs_IK1_R2","C24_R1_vs_IK24_R1","C24_R1_vs_IK24_R2","C24_R1_vs_IK4_R1","C24_R1_vs_IK4_R2","C24_R2_vs_C4_R1","C24_R2_vs_C4_R2","C24_R2_vs_IK0_B_R1","C24_R2_vs_IK0_B_R2","C24_R2_vs_IK0_R1","C24_R2_vs_IK0_R2","C24_R2_vs_IK1_R1","C24_R2_vs_IK1_R2","C24_R2_vs_IK24_R1","C24_R2_vs_IK24_R2","C24_R2_vs_IK4_R1","C24_R2_vs_IK4_R2","C4_R1_vs_C4_R2","C4_R1_vs_IK0_B_R1","C4_R1_vs_IK0_B_R2","C4_R1_vs_IK0_R1","C4_R1_vs_IK0_R2","C4_R1_vs_IK1_R1","C4_R1_vs_IK1_R2","C4_R1_vs_IK24_R1","C4_R1_vs_IK24_R2","C4_R1_vs_IK4_R1","C4_R1_vs_IK4_R2","C4_R2_vs_IK0_B_R1","C4_R2_vs_IK0_B_R2","C4_R2_vs_IK0_R1","C4_R2_vs_IK0_R2","C4_R2_vs_IK1_R1","C4_R2_vs_IK1_R2","C4_R2_vs_IK24_R1","C4_R2_vs_IK24_R2","C4_R2_vs_IK4_R1","C4_R2_vs_IK4_R2","IK0_B_R1_vs_IK0_B_R2","IK0_B_R1_vs_IK0_R1","IK0_B_R1_vs_IK0_R2","IK0_B_R1_vs_IK1_R1","IK0_B_R1_vs_IK1_R2","IK0_B_R1_vs_IK24_R1","IK0_B_R1_vs_IK24_R2","IK0_B_R1_vs_IK4_R1","IK0_B_R1_vs_IK4_R2","IK0_B_R2_vs_IK0_R1","IK0_B_R2_vs_IK0_R2","IK0_B_R2_vs_IK1_R1","IK0_B_R2_vs_IK1_R2","IK0_B_R2_vs_IK24_R1","IK0_B_R2_vs_IK24_R2","IK0_B_R2_vs_IK4_R1","IK0_B_R2_vs_IK4_R2","IK0_R1_vs_IK0_R2","IK0_R1_vs_IK1_R1","IK0_R1_vs_IK1_R2","IK0_R1_vs_IK24_R1","IK0_R1_vs_IK24_R2","IK0_R1_vs_IK4_R1","IK0_R1_vs_IK4_R2","IK0_R2_vs_IK1_R1","IK0_R2_vs_IK1_R2","IK0_R2_vs_IK24_R1","IK0_R2_vs_IK24_R2","IK0_R2_vs_IK4_R1","IK0_R2_vs_IK4_R2","IK1_R1_vs_IK1_R2","IK1_R1_vs_IK24_R1","IK1_R1_vs_IK24_R2","IK1_R1_vs_IK4_R1","IK1_R1_vs_IK4_R2","IK1_R2_vs_IK24_R1","IK1_R2_vs_IK24_R2","IK1_R2_vs_IK4_R1","IK1_R2_vs_IK4_R2","IK24_R1_vs_IK24_R2","IK24_R1_vs_IK4_R1","IK24_R1_vs_IK4_R2","IK24_R2_vs_IK4_R1","IK24_R2_vs_IK4_R2","IK4_R1_vs_IK4_R2"]}]}
</script>
<script type="ojs-define">
{"contents":[{"name":"consGC","value":["C0_B_R1_vs_C0_B_R2","C0_B_R1_vs_C0_R1","C0_B_R1_vs_C0_R2","C0_B_R1_vs_C1_R1","C0_B_R1_vs_C1_R2","C0_B_R1_vs_C24_R1","C0_B_R1_vs_C24_R2","C0_B_R1_vs_C4_R1","C0_B_R1_vs_C4_R2","C0_B_R1_vs_IK0_B_R1","C0_B_R1_vs_IK0_B_R2","C0_B_R1_vs_IK0_R1","C0_B_R1_vs_IK0_R2","C0_B_R1_vs_IK1_R1","C0_B_R1_vs_IK1_R2","C0_B_R1_vs_IK24_R1","C0_B_R1_vs_IK24_R2","C0_B_R1_vs_IK4_R1","C0_B_R1_vs_IK4_R2","C0_B_R2_vs_C0_R1","C0_B_R2_vs_C0_R2","C0_B_R2_vs_C1_R1","C0_B_R2_vs_C1_R2","C0_B_R2_vs_C24_R1","C0_B_R2_vs_C24_R2","C0_B_R2_vs_C4_R1","C0_B_R2_vs_C4_R2","C0_B_R2_vs_IK0_B_R1","C0_B_R2_vs_IK0_B_R2","C0_B_R2_vs_IK0_R1","C0_B_R2_vs_IK0_R2","C0_B_R2_vs_IK1_R1","C0_B_R2_vs_IK1_R2","C0_B_R2_vs_IK24_R1","C0_B_R2_vs_IK24_R2","C0_B_R2_vs_IK4_R1","C0_B_R2_vs_IK4_R2","C0_R1_vs_C0_R2","C0_R1_vs_C1_R1","C0_R1_vs_C1_R2","C0_R1_vs_C24_R1","C0_R1_vs_C24_R2","C0_R1_vs_C4_R1","C0_R1_vs_C4_R2","C0_R1_vs_IK0_B_R1","C0_R1_vs_IK0_B_R2","C0_R1_vs_IK0_R1","C0_R1_vs_IK0_R2","C0_R1_vs_IK1_R1","C0_R1_vs_IK1_R2","C0_R1_vs_IK24_R1","C0_R1_vs_IK24_R2","C0_R1_vs_IK4_R1","C0_R1_vs_IK4_R2","C0_R2_vs_C1_R1","C0_R2_vs_C1_R2","C0_R2_vs_C24_R1","C0_R2_vs_C24_R2","C0_R2_vs_C4_R1","C0_R2_vs_C4_R2","C0_R2_vs_IK0_B_R1","C0_R2_vs_IK0_B_R2","C0_R2_vs_IK0_R1","C0_R2_vs_IK0_R2","C0_R2_vs_IK1_R1","C0_R2_vs_IK1_R2","C0_R2_vs_IK24_R1","C0_R2_vs_IK24_R2","C0_R2_vs_IK4_R1","C0_R2_vs_IK4_R2","C1_R1_vs_C1_R2","C1_R1_vs_C24_R1","C1_R1_vs_C24_R2","C1_R1_vs_C4_R1","C1_R1_vs_C4_R2","C1_R1_vs_IK0_B_R1","C1_R1_vs_IK0_B_R2","C1_R1_vs_IK0_R1","C1_R1_vs_IK0_R2","C1_R1_vs_IK1_R1","C1_R1_vs_IK1_R2","C1_R1_vs_IK24_R1","C1_R1_vs_IK24_R2","C1_R1_vs_IK4_R1","C1_R1_vs_IK4_R2","C1_R2_vs_C24_R1","C1_R2_vs_C24_R2","C1_R2_vs_C4_R1","C1_R2_vs_C4_R2","C1_R2_vs_IK0_B_R1","C1_R2_vs_IK0_B_R2","C1_R2_vs_IK0_R1","C1_R2_vs_IK0_R2","C1_R2_vs_IK1_R1","C1_R2_vs_IK1_R2","C1_R2_vs_IK24_R1","C1_R2_vs_IK24_R2","C1_R2_vs_IK4_R1","C1_R2_vs_IK4_R2","C24_R1_vs_C24_R2","C24_R1_vs_C4_R1","C24_R1_vs_C4_R2","C24_R1_vs_IK0_B_R1","C24_R1_vs_IK0_B_R2","C24_R1_vs_IK0_R1","C24_R1_vs_IK0_R2","C24_R1_vs_IK1_R1","C24_R1_vs_IK1_R2","C24_R1_vs_IK24_R1","C24_R1_vs_IK24_R2","C24_R1_vs_IK4_R1","C24_R1_vs_IK4_R2","C24_R2_vs_C4_R1","C24_R2_vs_C4_R2","C24_R2_vs_IK0_B_R1","C24_R2_vs_IK0_B_R2","C24_R2_vs_IK0_R1","C24_R2_vs_IK0_R2","C24_R2_vs_IK1_R1","C24_R2_vs_IK1_R2","C24_R2_vs_IK24_R1","C24_R2_vs_IK24_R2","C24_R2_vs_IK4_R1","C24_R2_vs_IK4_R2","C4_R1_vs_C4_R2","C4_R1_vs_IK0_B_R1","C4_R1_vs_IK0_B_R2","C4_R1_vs_IK0_R1","C4_R1_vs_IK0_R2","C4_R1_vs_IK1_R1","C4_R1_vs_IK1_R2","C4_R1_vs_IK24_R1","C4_R1_vs_IK24_R2","C4_R1_vs_IK4_R1","C4_R1_vs_IK4_R2","C4_R2_vs_IK0_B_R1","C4_R2_vs_IK0_B_R2","C4_R2_vs_IK0_R1","C4_R2_vs_IK0_R2","C4_R2_vs_IK1_R1","C4_R2_vs_IK1_R2","C4_R2_vs_IK24_R1","C4_R2_vs_IK24_R2","C4_R2_vs_IK4_R1","C4_R2_vs_IK4_R2","IK0_B_R1_vs_IK0_B_R2","IK0_B_R1_vs_IK0_R1","IK0_B_R1_vs_IK0_R2","IK0_B_R1_vs_IK1_R1","IK0_B_R1_vs_IK1_R2","IK0_B_R1_vs_IK24_R1","IK0_B_R1_vs_IK24_R2","IK0_B_R1_vs_IK4_R1","IK0_B_R1_vs_IK4_R2","IK0_B_R2_vs_IK0_R1","IK0_B_R2_vs_IK0_R2","IK0_B_R2_vs_IK1_R1","IK0_B_R2_vs_IK1_R2","IK0_B_R2_vs_IK24_R1","IK0_B_R2_vs_IK24_R2","IK0_B_R2_vs_IK4_R1","IK0_B_R2_vs_IK4_R2","IK0_R1_vs_IK0_R2","IK0_R1_vs_IK1_R1","IK0_R1_vs_IK1_R2","IK0_R1_vs_IK24_R1","IK0_R1_vs_IK24_R2","IK0_R1_vs_IK4_R1","IK0_R1_vs_IK4_R2","IK0_R2_vs_IK1_R1","IK0_R2_vs_IK1_R2","IK0_R2_vs_IK24_R1","IK0_R2_vs_IK24_R2","IK0_R2_vs_IK4_R1","IK0_R2_vs_IK4_R2","IK1_R1_vs_IK1_R2","IK1_R1_vs_IK24_R1","IK1_R1_vs_IK24_R2","IK1_R1_vs_IK4_R1","IK1_R1_vs_IK4_R2","IK1_R2_vs_IK24_R1","IK1_R2_vs_IK24_R2","IK1_R2_vs_IK4_R1","IK1_R2_vs_IK4_R2","IK24_R1_vs_IK24_R2","IK24_R1_vs_IK4_R1","IK24_R1_vs_IK4_R2","IK24_R2_vs_IK4_R1","IK24_R2_vs_IK4_R2","IK4_R1_vs_IK4_R2"]}]}
</script>
<script type="ojs-define">
{"contents":[{"name":"deseq","value":["C_1_vs_C_0","C_24_vs_C_0","C_4_vs_C_0","IK_0_vs_C_0_D12","IK_0_vs_C_0_D1256","IK_0_vs_C_0","IK_1_vs_C_1","IK_24_vs_C_24","IK_4_vs_C_4"]}]}
</script>


<link rel="stylesheet" href="./assets/style.css">
</head>

<body>

<header id="title-block-header" class="quarto-title-block default toc-left page-columns page-full">
  <div class="quarto-title-banner page-columns page-full">
    <div class="quarto-title column-body">
      <h1 class="title">Analysis of human myeloid cell activation using ATAC sequencing</h1>
            <p class="subtitle lead">Project #5986</p>
                </div>
  </div>
    
  



  <div class="quarto-title-meta">

      <div>
      <div class="quarto-title-meta-heading">Author</div>
      <div class="quarto-title-meta-contents">
               <p>Vincent van Hoef </p>
            </div>
    </div>
      
      <div>
      <div class="quarto-title-meta-heading">Published</div>
      <div class="quarto-title-meta-contents">
        <p class="date">6/3/22</p>
      </div>
    </div>
      
    </div>
    
  
  </header><div id="quarto-content" class="page-columns page-rows-contents page-layout-article toc-left">
<div id="quarto-sidebar-toc-left" class="sidebar toc-left">
  <nav id="TOC" role="doc-toc">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#introduction" id="toc-introduction" class="nav-link active" data-scroll-target="#introduction">Introduction</a></li>
  <li><a href="#material-and-methods" id="toc-material-and-methods" class="nav-link" data-scroll-target="#material-and-methods">Material and Methods</a>
  <ul class="collapse">
  <li><a href="#preprocessing-pipeline" id="toc-preprocessing-pipeline" class="nav-link" data-scroll-target="#preprocessing-pipeline">Preprocessing Pipeline</a></li>
  </ul></li>
  <li><a href="#results" id="toc-results" class="nav-link" data-scroll-target="#results">Results</a>
  <ul class="collapse">
  <li><a href="#quality-control" id="toc-quality-control" class="nav-link" data-scroll-target="#quality-control">Quality control</a>
  <ul class="collapse">
  <li><a href="#overview" id="toc-overview" class="nav-link" data-scroll-target="#overview">Overview</a></li>
  <li><a href="#fragment-size" id="toc-fragment-size" class="nav-link" data-scroll-target="#fragment-size">Fragment size</a></li>
  <li><a href="#frip" id="toc-frip" class="nav-link" data-scroll-target="#frip">FRiP</a></li>
  <li><a href="#number-of-peaks" id="toc-number-of-peaks" class="nav-link" data-scroll-target="#number-of-peaks">Number of peaks</a></li>
  </ul></li>
  <li><a href="#normalization" id="toc-normalization" class="nav-link" data-scroll-target="#normalization">Normalization</a>
  <ul class="collapse">
  <li><a href="#overview-1" id="toc-overview-1" class="nav-link" data-scroll-target="#overview-1">Overview</a></li>
  <li><a href="#md-plots-and-fc-content" id="toc-md-plots-and-fc-content" class="nav-link" data-scroll-target="#md-plots-and-fc-content">MD plots and FC content</a></li>
  </ul></li>
  <li><a href="#pca" id="toc-pca" class="nav-link" data-scroll-target="#pca">PCA</a></li>
  <li><a href="#differential-expression" id="toc-differential-expression" class="nav-link" data-scroll-target="#differential-expression">Differential Expression</a></li>
  <li><a href="#gene-set-enrichment-analysis" id="toc-gene-set-enrichment-analysis" class="nav-link" data-scroll-target="#gene-set-enrichment-analysis">Gene set Enrichment Analysis</a>
  <ul class="collapse">
  <li><a href="#over-representation-analysis-ora" id="toc-over-representation-analysis-ora" class="nav-link" data-scroll-target="#over-representation-analysis-ora">Over-representation Analysis (ORA)</a></li>
  <li><a href="#gene-set-enrichment-analysis-1" id="toc-gene-set-enrichment-analysis-1" class="nav-link" data-scroll-target="#gene-set-enrichment-analysis-1">Gene Set Enrichment Analysis</a></li>
  <li><a href="#comparison-donor-pairs" id="toc-comparison-donor-pairs" class="nav-link" data-scroll-target="#comparison-donor-pairs">Comparison Donor Pairs</a></li>
  </ul></li>
  </ul></li>
  <li><a href="#data-responsibility" id="toc-data-responsibility" class="nav-link" data-scroll-target="#data-responsibility">Data Responsibility</a></li>
  <li><a href="#acknowledgements" id="toc-acknowledgements" class="nav-link" data-scroll-target="#acknowledgements">Acknowledgements</a></li>
  <li><a href="#closing-procedures" id="toc-closing-procedures" class="nav-link" data-scroll-target="#closing-procedures">Closing Procedures</a></li>
  </ul>
</nav>
</div>
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
</div>
<main class="content quarto-banner-title-block page-columns page-full" id="quarto-document-content">
<!-- Add icon library -->
<link rel="stylesheet" href="./assets/font-awesome-4.7.0/css/font-awesome.min.css">




<div class="cell">

</div>
<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target="#callout-1" aria-controls="callout-1" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Click here to see the version and updates of this report
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-1" class="callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Here you can see the status of the report. This will be updated as sections are added or improved.</p>
<ul>
<li>Version 1: [1/6/22] First version - several sections still missing due to offline server.</li>
<li>Version 1.1: [3/6/22] Added old donors, additional differential peaks usage, pathway enrichment and various other plots.</li>
</ul>
</div>
</div>
</div>
<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Note
</div>
</div>
<div class="callout-body-container callout-body">
<p>Please let me know if any of the visualizations or links seem to be erratic. We recently changed reporting tool and small issues might still be present.</p>
<p>You can leave comments on this report by opening the sidebar on the very right of this page. After signing up for Hypothes.is via the link at the top of the sidebar, commenting is quick and easy. Just highlight the section you want to remark upon and write your comment. Leave it in the public group (“Post to Public”) and I will be able to see it and update the text if necessary.</p>
</div>
</div>
<section id="introduction" class="level1">
<h1>Introduction</h1>
<p>This is the report for the analysis of project #5986. It includes the preprocessing, quality control and downstream analysis (i.e.&nbsp;differential expression and pathway enrichment analysis). The experiment consist of 2 patient-derived cell lines; each either knocked down for IRAK3 or control and followed during 4 timepoints: 0, 1, 4 and 24 hours. In addition 2 donors (donor 1 and 2) were analyzed at timepoint 0. So, a total of 20 samples to be analyzed:</p>
<table class="table">
<thead>
<tr class="header">
<th>Name</th>
<th style="text-align: center;">Treatment</th>
<th style="text-align: center;">Time</th>
<th style="text-align: center;">Donor</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>C0_R1</td>
<td style="text-align: center;">Control</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">5</td>
</tr>
<tr class="even">
<td>C0_R2</td>
<td style="text-align: center;">Control</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">6</td>
</tr>
<tr class="odd">
<td>C0_B_R1</td>
<td style="text-align: center;">Control</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">1</td>
</tr>
<tr class="even">
<td>C0_B_R2</td>
<td style="text-align: center;">Control</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">2</td>
</tr>
<tr class="odd">
<td>C1_R1</td>
<td style="text-align: center;">Control</td>
<td style="text-align: center;">1</td>
<td style="text-align: center;">5</td>
</tr>
<tr class="even">
<td>C1_R2</td>
<td style="text-align: center;">Control</td>
<td style="text-align: center;">1</td>
<td style="text-align: center;">6</td>
</tr>
<tr class="odd">
<td>C4_R1</td>
<td style="text-align: center;">Control</td>
<td style="text-align: center;">4</td>
<td style="text-align: center;">5</td>
</tr>
<tr class="even">
<td>C4_R2</td>
<td style="text-align: center;">Control</td>
<td style="text-align: center;">4</td>
<td style="text-align: center;">6</td>
</tr>
<tr class="odd">
<td>C24_R1</td>
<td style="text-align: center;">Control</td>
<td style="text-align: center;">24</td>
<td style="text-align: center;">5</td>
</tr>
<tr class="even">
<td>C24_R2</td>
<td style="text-align: center;">Control</td>
<td style="text-align: center;">24</td>
<td style="text-align: center;">6</td>
</tr>
<tr class="odd">
<td>IK0_R1</td>
<td style="text-align: center;">Knockdown</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">5</td>
</tr>
<tr class="even">
<td>IK0_R2</td>
<td style="text-align: center;">Knockdown</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">6</td>
</tr>
<tr class="odd">
<td>IK0_B_R1</td>
<td style="text-align: center;">Knockdown</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">1</td>
</tr>
<tr class="even">
<td>IK0_B_R2</td>
<td style="text-align: center;">Knockdown</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">2</td>
</tr>
<tr class="odd">
<td>IK1_R1</td>
<td style="text-align: center;">Knockdown</td>
<td style="text-align: center;">1</td>
<td style="text-align: center;">5</td>
</tr>
<tr class="even">
<td>IK1_R2</td>
<td style="text-align: center;">Knockdown</td>
<td style="text-align: center;">1</td>
<td style="text-align: center;">6</td>
</tr>
<tr class="odd">
<td>IK4_R1</td>
<td style="text-align: center;">Knockdown</td>
<td style="text-align: center;">4</td>
<td style="text-align: center;">5</td>
</tr>
<tr class="even">
<td>IK4_R2</td>
<td style="text-align: center;">Knockdown</td>
<td style="text-align: center;">4</td>
<td style="text-align: center;">6</td>
</tr>
<tr class="odd">
<td>IK24_R1</td>
<td style="text-align: center;">Knockdown</td>
<td style="text-align: center;">24</td>
<td style="text-align: center;">5</td>
</tr>
<tr class="even">
<td>IK24_R2</td>
<td style="text-align: center;">Knockdown</td>
<td style="text-align: center;">24</td>
<td style="text-align: center;">6</td>
</tr>
</tbody>
</table>
<p>The donors will serve as replicates to allow for a statistical analysis. The biggest interest is comparing the knockdown of IRAK3 versus the control for each time point. Also of interest is the time-dependent chromatin accesibility in the control cells after stimulation, so comparing C1, C4 and C24 versus C0.</p>
<p>A contract was set up with the following deliverables:</p>
<ul>
<li>Raw data transfer to Bianca project</li>
<li>Run best practice preprocessing pipeline</li>
<li>Pre- and post alignment quality control</li>
<li>Peak calling</li>
<li>Differential peak analysis + annotation</li>
<li>Pathway enrichment</li>
<li>Report with all results, tables and analysis code</li>
</ul>
</section>
<section id="material-and-methods" class="level1">
<h1>Material and Methods</h1>
<p>This is a description of the methods used in this analysis. For a potential later manuscript M&amp;M section, you can always contact NBIS for an up-to-date description of what ended up in the final manuscript, even after closing the project.</p>
<section id="preprocessing-pipeline" class="level2">
<h2 class="anchored" data-anchor-id="preprocessing-pipeline">Preprocessing Pipeline</h2>
<p>The data was delivered as standard fastq files containing the raw, unmapped read data. The data preprocessing was done by running the nf-core atacseq pipeline (v 1.2.2). Briefly, the following steps were performed:</p>
<ol type="1">
<li>Raw read QC</li>
<li>Adapter trimming</li>
<li>Alignment (BWA)</li>
<li>Mark duplicates</li>
<li>Merge alignments from multiple libraries of the same sample
<ul>
<li>Re-mark duplicates</li>
<li>Filtering to remove:
<ul>
<li>reads mapping to mitochondrial DNA</li>
<li>reads mapping to blacklisted regions</li>
<li>reads that are marked as duplicates</li>
<li>reads that arent marked as primary alignments</li>
<li>reads that are unmapped</li>
<li>reads that map to multiple locations</li>
<li>reads containing &gt; 4 mismatches</li>
<li>reads that are soft-clipped</li>
<li>reads that have an insert size &gt; 2kb</li>
<li>reads that map to different chromosomes</li>
<li>reads that arent in FR orientation</li>
<li>reads where only one read of the pair fails the above criteria</li>
</ul></li>
<li>Alignment-level QC and estimation of library complexity</li>
<li>Create normalised bigWig files scaled to 1 million mapped reads</li>
<li>Generate gene-body meta-profile from bigWig files</li>
<li>Calculate genome-wide enrichment</li>
<li>Call broad peaks (MACS2)</li>
<li>Annotate peaks relative to gene features</li>
<li>Create consensus peakset across all samples and create tabular file to aid in the filtering of the data</li>
<li>Count reads in consensus peaks (featureCounts)</li>
<li>Differential accessibility analysis, PCA and clustering</li>
<li>Generate ATAC-seq specific QC html report</li>
</ul></li>
<li>Create IGV session file containing bigWig tracks, peaks and differential sites for data visualisation (IGV).</li>
<li>Present QC for raw read, alignment, peak-calling and differential accessibility results (ataqv, MultiQC, R)</li>
</ol>
</section>
</section>
<section id="results" class="level1 page-columns page-full">
<h1>Results</h1>
<section id="quality-control" class="level2">
<h2 class="anchored" data-anchor-id="quality-control">Quality control</h2>
<p>Modern high throughput sequencers can generate hundreds of millions of sequences in a single run. Before analysing this sequences to draw biological conclusions you should always perform some simple quality control checks to ensure that the raw data looks good and there are no problems or biases in your data which may affect how you can usefully use it. Two very useful quality control softwares were run on the data: multiQC and ataqv. These contain many high-quality plots; most of them are interactive and with relevant help sections to help interpretation. Most figures can be filtered and downloaded as needed. If the figures are too small to see, please right-click and “Open Frame in New Tab”</p>
<section id="overview" class="level3">
<h3 class="anchored" data-anchor-id="overview">Overview</h3>
<div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-1-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-1" role="tab" aria-controls="tabset-1-1" aria-selected="true">MultiQC</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-2" role="tab" aria-controls="tabset-1-2" aria-selected="false">ataqv</a></li></ul>
<div class="tab-content">
<div id="tabset-1-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-1-1-tab">
<p>MultiQC is a reporting tool that parses summary statistics from results and log files generated by other bioinformatics tools. The LIB sections are done for each library separately - so for each technical replicate and each forward and reverse read collection (64 samples in total). MERGED LIB are for the 16 biological samples (so, technical replicates merged and looking at fragments after mapping the forward and reverse read pairs). The QC tools summarized here are:</p>
<ul>
<li>FastQC: raw quality measures on base and read level (before and after trimming adapters)</li>
<li>Cutadapt: QC of trimming step</li>
<li>SAMTools: mapping statistics</li>
<li>Preseq: library complexity</li>
<li>Picard: diverse statistics (of special interest here is the Insert size)</li>
<li>DeepTools: fingerprint plot and read distribution profile</li>
<li>MACS2: peak count and frip score</li>
<li>HOMER: peak annotation proportion</li>
<li>featureCounts: proportion of reads assigned to known genetic features</li>
<li>DESEeq2: PCA and sample similarity</li>
</ul>
<div id="fig-multiqc" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<iframe width="560" height="615" src="./Results/multiqc/broadPeak/multiqc_report.html">
</iframe>
<p></p><figcaption aria-hidden="true" class="figure-caption">Figure 1: MultiQC Report.</figcaption><p></p>
</figure>
</div>
</div>
<div id="tabset-1-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-2-tab">
<p>ataqv makes some ATAC specific quality plots. Again, below is an interactive report showing these plots. Help for the specific plots can be found by clicking on the question mark in each figure.</p>
<div id="fig-ataqv" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<iframe width="560" height="415" src="./Results/bwa/mergedLibrary/ataqv/broadPeak/html/index.html">
</iframe>
<p></p><figcaption aria-hidden="true" class="figure-caption">Figure 2: Ataqv Report.</figcaption><p></p>
</figure>
</div>
</div>
</div>
</div>
<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Note
</div>
</div>
<div class="callout-body-container callout-body">
<p>To summarize, the quality of the data looks good to excellent. There are no outlier samples that should be removed before the downstream analysis nor seem there be to be systematic differences between the different donors, time points or treatment.</p>
</div>
</div>
<p>Below, a few ATAC-specific quality plots are highlighted.</p>
</section>
<section id="fragment-size" class="level3">
<h3 class="anchored" data-anchor-id="fragment-size">Fragment size</h3>
<p>Typically, a successful ATAC-seq experiment should generate a fragment size distribution plot with decreasing and periodical peaks corresponding to the nucleosome-free regions (NFR) (&lt; 100bp) and mono-, di-, and tri-nucleosomes (~200, 400, 600bp, respectively). All samples have a fragment size distribution showing the expected peaks.</p>
<div class="cell">

</div>
<div class="cell">
<div class="sourceCode cell-code hidden" id="cb1" data-startfrom="196" data-source-offset="-0"><pre class="sourceCode js code-with-copy"><code class="sourceCode javascript" style="counter-reset: source-line 195;"><span id="cb1-196"><a href="#cb1-196" aria-hidden="true" tabindex="-1"></a>viewof sample_sel <span class="op">=</span> Inputs<span class="op">.</span><span class="fu">select</span>(insert_hist<span class="op">,</span> { <span class="dt">label</span><span class="op">:</span> <span class="st">"Choose Sample"</span> })</span>
<span id="cb1-197"><a href="#cb1-197" aria-hidden="true" tabindex="-1"></a><span class="fu">html</span><span class="vs">`&lt;p&gt;&lt;/p&gt;`</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div>
<div id="ojs-cell-1-1" data-nodetype="declaration">

</div>
</div>
</div>
<div class="cell-output cell-output-display">
<div>
<div id="ojs-cell-1-2" data-nodetype="expression">

</div>
</div>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code hidden" id="cb2" data-startfrom="205" data-source-offset="-65"><pre class="sourceCode js code-with-copy"><code class="sourceCode javascript" style="counter-reset: source-line 204;"><span id="cb2-205"><a href="#cb2-205" aria-hidden="true" tabindex="-1"></a><span class="fu">html</span><span class="vs">`&lt;iframe src="./Results/bwa/mergedLibrary/picard_metrics/pdf/</span><span class="sc">${</span>sample_sel<span class="sc">}</span><span class="vs">.mLb.clN.CollectMultipleMetrics.insert_size_histogram.pdf" width:100% height:600px&gt;&lt;/iframe&gt;`</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div id="fig-insert-hist" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<div id="ojs-cell-2" data-nodetype="expression">

</div>
<p></p><figcaption aria-hidden="true" class="figure-caption">Figure 3: Insert size histogram.</figcaption><p></p>
</figure>
</div>
</div>
</section>
<section id="frip" class="level3">
<h3 class="anchored" data-anchor-id="frip">FRiP</h3>
<p>The fraction of reads in called peaks is another important QC meaasure. Recommendations are to preferably have more than 20% of reads in peak regions, so a FRip &gt; 0.2. This is the case for most samples here.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="./Results/qc/frip_score.svg" class="img-fluid figure-img"></p>
<p></p><figcaption aria-hidden="true" class="figure-caption">Fraction of Reads in Peaks</figcaption><p></p>
</figure>
</div>
</section>
<section id="number-of-peaks" class="level3">
<h3 class="anchored" data-anchor-id="number-of-peaks">Number of peaks</h3>
<p>Peaks were only called when present in both replicates; this to reduce most of the donor specific effects. In total, so combining all peaks found in both replicates across all samples, the number of peaks is 82704. Below you can see an UpSet plot of how these are distributed among the samples.</p>
<div id="fig-peaks" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<iframe width="100%" height="600" src="./Results/bwa/mergedLibrary/macs/broadPeak/consensus/consensus_peaks.mLb.clN.boolean.intersect.plot.pdf">
</iframe>
<p></p><figcaption aria-hidden="true" class="figure-caption">Figure 4: Peak Count.</figcaption><p></p>
</figure>
</div>
</section>
</section>
<section id="normalization" class="level2">
<h2 class="anchored" data-anchor-id="normalization">Normalization</h2>
<p>Several technical issues can result in different read depths and distributions across samples. Examples of these issues are sample quality, sample amount, sequencing depth, GC content, … If these issues differentially affect samples, this can result in spurious differentially expressed peaks; i.e one could have both false positive and false negative results.</p>
<section id="overview-1" class="level3">
<h3 class="anchored" data-anchor-id="overview-1">Overview</h3>
<p>Below are two plots showing the need for normalization. Left you can see an <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0191629">RLE plot</a> which is a powerful tool for visualizing unwanted variation in high-dimensional data. Because of their ability to detect unwanted variation, RLE plots are particularly useful for assessing whether a normalization procedure, i.e.&nbsp;a procedure aimed at removing unwanted variation, has been successful: a “bad” plot would suggest a failure to normalize. In a “good” plot the boxplots are roughly centered at zero. As you can see in the pre-normalization panel, there is variation across the samples that should probably be removed.</p>
<p>On the right, you can see curves displaying the smoothed gene or peak counts per sample in function of GC content. GC content can be of particular importance in ATAC seq because the enzymatic cut by Tn5 will introduce bias due to binding preference, which is in itself associated with GC content. The accessibility count of a particular genomic region is thus associated with its GC-content. However, the slope and shape of the curves may differ between samples, which indicates that GC-content effects are sample-specific and can therefore bias between-sample comparisons. One might initially think that, because DA analyses involve comparing read counts between samples for a given genomic region with a fixed GC-content, GC-content effects would cancel out. However, because of their sample-specificity, GC-content effects also impact log-fold-changes comparing accessibility between samples for a given region. In the pre-normalization plot below there is indeed some evidence for a different average peak count across the samples.</p>
<p>The two plots show the need for normalization. Many methods are available (and were indeed tried during the analysis) but here the results are shown for FQ-FQ normalization. This is a GC aware normalization method, based on two rounds of full-quantile normalization. First, FQ normalization within a sample across bins of features (e.g., peaks or genes) with similar GC-content and, subsequently, FQ normalization between samples. FQ-FQ and other normalization methods for ATAC-seq have recently been described in <a href="https://www.biorxiv.org/content/10.1101/2021.01.26.428252v1.full">this paper</a>. Then FQ normalization panel below show this normalization indeed makes the samples more comparable.</p>
<div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-2-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-1" role="tab" aria-controls="tabset-2-1" aria-selected="true">Before Normalization</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-2-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-2" role="tab" aria-controls="tabset-2-2" aria-selected="false">Post Normalization</a></li></ul>
<div class="tab-content">
<div id="tabset-2-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-2-1-tab">
<div class="quarto-layout-panel">
<figure class="figure">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-pre-rle" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="https://uppsala.box.com/shared/static/9k05fwnjvqh4xc88inwu5ufmx5m3p6nr.png" class="img-fluid figure-img" data-ref-parent="fig-pre"></p>
<p></p><figcaption aria-hidden="true" class="figure-caption">(a) RLE Boxplots</figcaption><p></p>
</figure>
</div>
</div>
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-pre-gc" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="https://uppsala.box.com/shared/static/93pbgj9l0k0tt9clc8h1ykvimmeuout1.png" class="img-fluid figure-img" data-ref-parent="fig-pre"></p>
<p></p><figcaption aria-hidden="true" class="figure-caption">(b) Peak counts in function of GC-content</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p></p><figcaption aria-hidden="true" class="figure-caption">Figure 5: Before Normalization</figcaption><p></p>
</figure>
</div>
</div>
<div id="tabset-2-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-2-2-tab">
<div class="quarto-layout-panel">
<figure class="figure">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-post-rle" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="https://uppsala.box.com/shared/static/a9druk2ggqw0ijvob0u81jkd82fxf7i7.png" class="img-fluid figure-img" data-ref-parent="fig-post"></p>
<p></p><figcaption aria-hidden="true" class="figure-caption">(a) RLE Boxplots</figcaption><p></p>
</figure>
</div>
</div>
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-post-gc" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="https://uppsala.box.com/shared/static/m0zny3g7aet0wdzyq3agjlvxl1zw0e0k.png" class="img-fluid figure-img" data-ref-parent="fig-post"></p>
<p></p><figcaption aria-hidden="true" class="figure-caption">(b) Peak counts in function of GC-content</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p></p><figcaption aria-hidden="true" class="figure-caption">Figure 6: After Normalization</figcaption><p></p>
</figure>
</div>
</div>
</div>
</div>
</section>
<section id="md-plots-and-fc-content" class="level3">
<h3 class="anchored" data-anchor-id="md-plots-and-fc-content">MD plots and FC content</h3>
<p>Below you can see the effect of FQ-FQ normalization for a selection of group comparisons. On the left are the pre-normalization plots, and on the the right after normalization.</p>
<p>MD plots are shown in <a href="#fig-md">Figure&nbsp;7</a>. MD plots depict global patterns of measurements compared between two samples, where each tested genomic feature is quantified by the difference between two groups as the y-axis (D) and average logged read count as the x-axis (M). Each dot is a single interval/peak. For well normalized data, one would expect most peaks not to differ between groups (if we assume most peaks are similar between the two groups). As visible in the plots below, after FQ-FQ normalization (right plot) the bulk of the peaks centers around 0 and most systematic differences between groups has been normalized.</p>
<div class="cell">

</div>
<div class="cell">
<div class="sourceCode cell-code hidden" id="cb3" data-startfrom="276" data-source-offset="-0"><pre class="sourceCode js code-with-copy"><code class="sourceCode javascript" style="counter-reset: source-line 275;"><span id="cb3-276"><a href="#cb3-276" aria-hidden="true" tabindex="-1"></a>viewof treat_sel <span class="op">=</span> Inputs<span class="op">.</span><span class="fu">select</span>(contrast<span class="op">,</span> { <span class="dt">label</span><span class="op">:</span> <span class="st">"Choose Contrast"</span> })</span>
<span id="cb3-277"><a href="#cb3-277" aria-hidden="true" tabindex="-1"></a><span class="fu">html</span><span class="vs">`&lt;p&gt;&lt;/p&gt;`</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div>
<div id="ojs-cell-3-1" data-nodetype="declaration">

</div>
</div>
</div>
<div class="cell-output cell-output-display">
<div>
<div id="ojs-cell-3-2" data-nodetype="expression">

</div>
</div>
</div>
</div>
<div class="quarto-layout-panel">
<figure class="figure">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="cell quarto-layout-cell" style="flex-basis: 50.0%;justify-content: center;">
<div class="sourceCode cell-code hidden" id="cb4" data-startfrom="283" data-source-offset="-0"><pre class="sourceCode js code-with-copy"><code class="sourceCode javascript" style="counter-reset: source-line 282;"><span id="cb4-283"><a href="#cb4-283" aria-hidden="true" tabindex="-1"></a><span class="fu">md</span><span class="vs">`![Before Normalization](./Results/EDA/MD/pre_norm/</span><span class="sc">${</span>treat_sel<span class="sc">}</span><span class="vs">_md.png)`</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div id="ojs-cell-4" data-nodetype="expression">

</div>
</div>
</div>
<div class="cell quarto-layout-cell" style="flex-basis: 50.0%;justify-content: center;">
<div class="sourceCode cell-code hidden" id="cb5" data-startfrom="287" data-source-offset="-0"><pre class="sourceCode js code-with-copy"><code class="sourceCode javascript" style="counter-reset: source-line 286;"><span id="cb5-287"><a href="#cb5-287" aria-hidden="true" tabindex="-1"></a><span class="fu">md</span><span class="vs">`![After Normalization](./Results/EDA/MD/post_norm/</span><span class="sc">${</span>treat_sel<span class="sc">}</span><span class="vs">_md.png)`</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div id="ojs-cell-5" data-nodetype="expression">

</div>
</div>
</div>
</div>
<p></p><figcaption aria-hidden="true" class="figure-caption">Figure 7: MD Plots.</figcaption><p></p>
</figure>
</div>
<p>In <a href="#fig-gc-con">Figure&nbsp;8</a>, stratified boxplots of the log fold change of read counts are shown. One would expect most log fold changes to be centered around zero and no systematic shifts associated with the gc content.</p>
<div class="cell">

</div>
<div class="cell">
<div class="sourceCode cell-code hidden" id="cb6" data-startfrom="302" data-source-offset="-0"><pre class="sourceCode js code-with-copy"><code class="sourceCode javascript" style="counter-reset: source-line 301;"><span id="cb6-302"><a href="#cb6-302" aria-hidden="true" tabindex="-1"></a>viewof consGC_sel <span class="op">=</span> Inputs<span class="op">.</span><span class="fu">select</span>(consGC<span class="op">,</span> { <span class="dt">label</span><span class="op">:</span> <span class="st">"Choose Contrast"</span> })</span>
<span id="cb6-303"><a href="#cb6-303" aria-hidden="true" tabindex="-1"></a><span class="fu">html</span><span class="vs">`&lt;p&gt;&lt;/p&gt;`</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div>
<div id="ojs-cell-6-1" data-nodetype="declaration">

</div>
</div>
</div>
<div class="cell-output cell-output-display">
<div>
<div id="ojs-cell-6-2" data-nodetype="expression">

</div>
</div>
</div>
</div>
<div class="quarto-layout-panel">
<figure class="figure">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="cell quarto-layout-cell" style="flex-basis: 50.0%;justify-content: center;">
<div class="sourceCode cell-code hidden" id="cb7" data-startfrom="309" data-source-offset="-0"><pre class="sourceCode js code-with-copy"><code class="sourceCode javascript" style="counter-reset: source-line 308;"><span id="cb7-309"><a href="#cb7-309" aria-hidden="true" tabindex="-1"></a><span class="fu">md</span><span class="vs">`![Before Normalization](./Results/EDA/GC_Content/pre_norm/</span><span class="sc">${</span>consGC_sel<span class="sc">}</span><span class="vs">_LFC_gc_bin.png)`</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div id="ojs-cell-7" data-nodetype="expression">

</div>
</div>
</div>
<div class="cell quarto-layout-cell" style="flex-basis: 50.0%;justify-content: center;">
<div class="sourceCode cell-code hidden" id="cb8" data-startfrom="313" data-source-offset="-0"><pre class="sourceCode js code-with-copy"><code class="sourceCode javascript" style="counter-reset: source-line 312;"><span id="cb8-313"><a href="#cb8-313" aria-hidden="true" tabindex="-1"></a><span class="fu">md</span><span class="vs">`![After Normalization](./Results/EDA/GC_Content/post_norm/</span><span class="sc">${</span>consGC_sel<span class="sc">}</span><span class="vs">_LFC_gc_bin.png)`</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div id="ojs-cell-8" data-nodetype="expression">

</div>
</div>
</div>
</div>
<p></p><figcaption aria-hidden="true" class="figure-caption">Figure 8: LogFC Boxplots binned for GC content.</figcaption><p></p>
</figure>
</div>
<p>Again, normalization greatly diminishes the biases affecting the peak counts.</p>
</section>
</section>
<section id="pca" class="level2">
<h2 class="anchored" data-anchor-id="pca">PCA</h2>
<p>Using <a href="http://setosa.io/ev/principal-component-analysis">Principal Component Analysis</a> (PCA), the structure in the data was visualized by projecting the samples on a two-dimensional graph using the two first principal components (PC). These components are linear combinations of the peaks that explain the most variation between those samples. These kind of plots can be used to examine the samples for outliers, sample swaps, batch effects and other relationships. When normalization successfully removed technical or batch artefacts, the relative distances between samples should be biologically interpretable. A plot of PC1 vs PC2 using the 500 most variable peaks and colored by their experimental group is shown below. The different time points do seem to cluster in this plot, with timepoints 1 and 4 more similar, separate from both timepoint 0 and 24. Knockdown samples are not easily distinguishable from control samples, indicating they are very alike. There does seem to be a small donor effect along PC2 (might be worth taking into account during DE…). From this PCA plot one can infer differences exist between the time points and that there do not appear to be major outliers or obvious sample swaps between time points.</p>
<div id="fig-pca" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="https://uppsala.box.com/shared/static/hwrbd1matqw7ycmo59zmxk6q3m4mnm2i.png" class="img-fluid figure-img"></p>
<p></p><figcaption aria-hidden="true" class="figure-caption">Figure 9: Principal Component Plot</figcaption><p></p>
</figure>
</div>
</section>
<section id="differential-expression" class="level2">
<h2 class="anchored" data-anchor-id="differential-expression">Differential Expression</h2>
<p>An important analysis question is the quantification and statistical inference of systematic changes between conditions, as compared to within-condition variability. The package DESeq2 provides methods to test for differential expression by use of negative binomial generalized linear models; the estimates of dispersion and logarithmic fold changes incorporate data-driven prior distributions.</p>
<p>The contrasts that will be compared are:</p>
<ul>
<li>C_1 vs C_0</li>
<li>C_4 vs C_0</li>
<li>C_24 vs C_0</li>
<li>IK_0 vs C_0</li>
<li>IK_1 vs C_1</li>
<li>IK_4 vs C_4</li>
<li>IK_24 vs C_24</li>
<li>IK_0 vs C_0 (Donor 1 and 2 only)</li>
<li>IK_0 vs C_0 (all 4 donors together)</li>
</ul>
<p>The design formula is normalized_counts ~ donor + treatment_group. This means we control for donor-specific effects.</p>
<p><em>Multiple test correction</em></p>
<p>Note that we have pvalues and p-adjusted values in the output. Which should we use to identify significantly differentially expressed genes?</p>
<p>If we used the p-value directly from with a significance cut-off of p &lt; 0.05, that means there is a 5% chance it is a false positives. Each p-value is the result of a single test (single peak). The more peaks we test, the more we inflate the false positive rate. This is the multiple testing problem. For example, if we test 20,000 peaks for differential expression, at p &lt; 0.05 we would expect to find 1,000 peaks by chance. If we found 3000 peaks to be differentially expressed total, roughly one third of our genes are false positives. We would not want to sift through our “significant” peaks to identify which ones are true positives.</p>
<p>DESeq2 helps reduce the number of peaks tested by removing those genes unlikely to be significantly DE prior to testing, such as those with low number of counts and outlier samples (gene-level QC). However, we still need to correct for multiple testing to reduce the number of false positives. In DESeq2, the p-values attained are corrected for multiple testing using the Benjamini and Hochberg method by default. <strong>These p-adjusted values should be used to determine significant genes.</strong></p>
<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Note
</div>
</div>
<div class="callout-body-container callout-body">
<p>So what does FDR &lt; 0.05 mean? By setting the FDR cutoff to &lt; 0.05, we’re saying that the proportion of false positives we expect amongst our differentially expressed genes is 5%. For example, if you call 500 peaks as differentially expressed with an FDR cutoff of 0.05, you expect 25 of them to be false positives.</p>
</div>
</div>
<p>A volcano plot of the results for each contrast can be seen below. Peaks are colored red if their adjusted p value is smaller than 0.05 and the log2 Fold Change is larger than 1 or smaller than -1. Annotation of the peaks is done using HOMER (description below).</p>
<div class="cell">

</div>
<div class="cell">
<div class="sourceCode cell-code hidden" id="cb9" data-startfrom="368" data-source-offset="-0"><pre class="sourceCode js code-with-copy"><code class="sourceCode javascript" style="counter-reset: source-line 367;"><span id="cb9-368"><a href="#cb9-368" aria-hidden="true" tabindex="-1"></a>viewof deseq_sel <span class="op">=</span> Inputs<span class="op">.</span><span class="fu">select</span>(deseq<span class="op">,</span> { <span class="dt">label</span><span class="op">:</span> <span class="st">"Choose Contrast"</span> })</span>
<span id="cb9-369"><a href="#cb9-369" aria-hidden="true" tabindex="-1"></a><span class="fu">html</span><span class="vs">`&lt;p&gt;&lt;/p&gt;`</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div>
<div id="ojs-cell-9-1" data-nodetype="declaration">

</div>
</div>
</div>
<div class="cell-output cell-output-display">
<div>
<div id="ojs-cell-9-2" data-nodetype="expression">

</div>
</div>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code hidden" id="cb10" data-startfrom="374" data-source-offset="-0"><pre class="sourceCode js code-with-copy"><code class="sourceCode javascript" style="counter-reset: source-line 373;"><span id="cb10-374"><a href="#cb10-374" aria-hidden="true" tabindex="-1"></a><span class="fu">md</span><span class="vs">`![Volcano Plot](./Results/DESeq2/</span><span class="sc">${</span>deseq_sel<span class="sc">}</span><span class="vs">/</span><span class="sc">${</span>deseq_sel<span class="sc">}</span><span class="vs">_volcano.png)`</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div id="ojs-cell-10" data-nodetype="expression">

</div>
</div>
</div>
<p>Full results can be downloaded from here.</p>
<iframe src="https://uppsala.app.box.com/embed/s/mvf0fuxlvyrxp7d8a8r7eap1n5crzgng?sortColumn=date&amp;view=list" width="500" height="400" frameborder="0" allowfullscreen="" webkitallowfullscreen="" msallowfullscreen="">
</iframe>
<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
How Basic Annotation Works
</div>
</div>
<div class="callout-body-container callout-body">
<p>The process of annotating peaks/regions using HOMER is divided into two primary parts. The first determines the distance to the nearest TSS and assigns the peak to that gene. The second determines the genomic annotation of the region occupied by the center of the peak/region.</p>
<p><em>Distance to the nearest TSS</em></p>
<p>By default, it uses the positions of RefSeq transcription start sites to determine the closest TSS, reporting the distance (negative values mean upstream of the TSS, positive values mean downstream), and various annotation information linked to locus including alternative identifiers (unigene, entrez gene, ensembl, gene symbol etc.).</p>
<p><em>Genomic Annotation</em></p>
<p>To annotate the location of a given peak in terms of important genomic features, HOMER calls a separate program (assignGenomeAnnotation) to efficiently assign peaks to one of millions of possible annotations genome wide. Two types of output are provided. The first is “Basic Annotation” that includes whether a peak is in the TSS (transcription start site), TTS (transcription termination site), Exon (Coding), 5’ UTR Exon, 3’ UTR Exon, Intronic, or Intergenic, which are common annotations that many researchers are interested in. A second round of “Detailed Annotation” also includes more detailed annotation, also considering repeat elements and CpG islands. Since some annotation overlap, a priority is assign based on the following (in case of ties it’s random [i.e.&nbsp;if there are two overlapping repeat element annotations]):</p>
<ul>
<li>TSS (by default defined from -1kb to +100bp)</li>
<li>TTS (by default defined from -100 bp to +1kb)</li>
<li>CDS Exons</li>
<li>5’ UTR Exons</li>
<li>3’ UTR Exons</li>
<li>**CpG Islands</li>
<li>**Repeats</li>
<li>Introns</li>
<li>Intergenic</li>
</ul>
<p>** Only applicable for the “Detailed Annotation”.</p>
<p>Although HOMER doesn’t allow you to explicitly change the definition of the region that is the TSS (-1kb to +100bp), you can “do it yourself” by sorting the annotation output in EXCEL by the “Distance to nearest TSS” column, and selecting those within the range you are interested in.</p>
</div>
</div>
</section>
<section id="gene-set-enrichment-analysis" class="level2 page-columns page-full">
<h2 class="anchored" data-anchor-id="gene-set-enrichment-analysis">Gene set Enrichment Analysis</h2>
<p>An essential method to make biological sense of the peak/region data is pathway or functional enrichment analysis. Single differentially expressed peaks might not be so informative in and of themselves but if multiple peaks would regulate the expression of multiple genes that are linked to the same biological function, this might be an indication of a real biological effect. The difficulty in doing this using peak based data is that the link between peak and gene is not always clear. There is no real way for example to link a peak in an intergenic region to a specific gene, without extensive experimental work. As described above, the different peaks were annotated using HOMER, so a natural way to increase our confidence in the gene association of the different peaks is by subsetting all peaks to just the ones around known TSS and then selecting the closest gene. This is described for example in <a href="https://rockefelleruniversity.github.io/RU_ATACseq/">this tutorial</a>. For the pathway enrichment in this report, we only used the peaks located 1000bp upstream or 1000bp downstream of the TSS.</p>
<section id="over-representation-analysis-ora" class="level3 page-columns page-full">
<h3 class="anchored" data-anchor-id="over-representation-analysis-ora">Over-representation Analysis (ORA)</h3>
<p>Over-representation (or enrichment) analysis is a statistical method that determines whether genes from pre-defined sets (i.e those belonging to a specific GO term or KEGG pathway) are present more than would be expected (over-represented) in a subset of your data. In this case this subset of the data are the genes with a TSS covered by a peak (+/- 1000bp) with an FDR &lt; 0.05 and a log2FoldChange &gt; 0 for the “more accessible TSS” and Log2FoldChange &lt; 0 for the “Less accesible TSS”. The pre-defined gene sets tested here are the Biological Product terms of the GO annotation or the pathways in the KEGG repository.</p>
<p>The results are collected in interactive html reports. Clicking on a term will show which specific genes are in our dataset. Figures and tables can be downloaded.</p>
<div class="cell">
<div class="sourceCode cell-code hidden" id="cb11" data-startfrom="425" data-source-offset="-0"><pre class="sourceCode js code-with-copy"><code class="sourceCode javascript" style="counter-reset: source-line 424;"><span id="cb11-425"><a href="#cb11-425" aria-hidden="true" tabindex="-1"></a>viewof ora_sel <span class="op">=</span> Inputs<span class="op">.</span><span class="fu">select</span>([<span class="st">"C_1_vs_C_0"</span><span class="op">,</span> <span class="st">"C_4_vs_C_0"</span><span class="op">,</span> <span class="st">"C_24_vs_C_0"</span>]<span class="op">,</span> { <span class="dt">label</span><span class="op">:</span> <span class="st">"Choose Contrast"</span> })</span>
<span id="cb11-426"><a href="#cb11-426" aria-hidden="true" tabindex="-1"></a><span class="fu">html</span><span class="vs">`&lt;p&gt;&lt;/p&gt;`</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div>
<div id="ojs-cell-11-1" data-nodetype="declaration">

</div>
</div>
</div>
<div class="cell-output cell-output-display">
<div>
<div id="ojs-cell-11-2" data-nodetype="expression">

</div>
</div>
</div>
</div>
<div class="panel-tabset column-body-outset-right">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-3-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-3-1" role="tab" aria-controls="tabset-3-1" aria-selected="true">GO Up</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-3-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-3-2" role="tab" aria-controls="tabset-3-2" aria-selected="false">GO Down</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-3-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-3-3" role="tab" aria-controls="tabset-3-3" aria-selected="false">KEGG Up</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-3-4-tab" data-bs-toggle="tab" data-bs-target="#tabset-3-4" role="tab" aria-controls="tabset-3-4" aria-selected="false">KEGG Down</a></li></ul>
<div class="tab-content">
<div id="tabset-3-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-3-1-tab">
<div class="cell">
<div class="sourceCode cell-code hidden" id="cb12" data-startfrom="434" data-source-offset="-0"><pre class="sourceCode js code-with-copy"><code class="sourceCode javascript" style="counter-reset: source-line 433;"><span id="cb12-434"><a href="#cb12-434" aria-hidden="true" tabindex="-1"></a><span class="fu">html</span><span class="vs">`&lt;iframe src="./Results/DESeq2/</span><span class="sc">${</span>ora_sel<span class="sc">}</span><span class="vs">/Pathway_Enrichment/Project_</span><span class="sc">${</span>ora_sel<span class="sc">}</span><span class="vs">_go_up/Report_</span><span class="sc">${</span>ora_sel<span class="sc">}</span><span class="vs">_go_up.html" width:100% height:1200px&gt;&lt;/iframe&gt;`</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div id="ojs-cell-12" data-nodetype="expression">

</div>
</div>
</div>
</div>
<div id="tabset-3-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-3-2-tab">
<div class="cell">
<div class="sourceCode cell-code hidden" id="cb13" data-startfrom="440" data-source-offset="-0"><pre class="sourceCode js code-with-copy"><code class="sourceCode javascript" style="counter-reset: source-line 439;"><span id="cb13-440"><a href="#cb13-440" aria-hidden="true" tabindex="-1"></a><span class="fu">html</span><span class="vs">`&lt;iframe src="./Results/DESeq2/</span><span class="sc">${</span>ora_sel<span class="sc">}</span><span class="vs">/Pathway_Enrichment/Project_</span><span class="sc">${</span>ora_sel<span class="sc">}</span><span class="vs">_go_down/Report_</span><span class="sc">${</span>ora_sel<span class="sc">}</span><span class="vs">_go_down.html" width:100% height:1200px&gt;&lt;/iframe&gt;`</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div id="ojs-cell-13" data-nodetype="expression">

</div>
</div>
</div>
</div>
<div id="tabset-3-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-3-3-tab">
<div class="cell">
<div class="sourceCode cell-code hidden" id="cb14" data-startfrom="446" data-source-offset="-0"><pre class="sourceCode js code-with-copy"><code class="sourceCode javascript" style="counter-reset: source-line 445;"><span id="cb14-446"><a href="#cb14-446" aria-hidden="true" tabindex="-1"></a><span class="fu">html</span><span class="vs">`&lt;iframe src="./Results/DESeq2/</span><span class="sc">${</span>ora_sel<span class="sc">}</span><span class="vs">/Pathway_Enrichment/Project_</span><span class="sc">${</span>ora_sel<span class="sc">}</span><span class="vs">_kegg_up/Report_</span><span class="sc">${</span>ora_sel<span class="sc">}</span><span class="vs">_kegg_up.html" width:100% height:1200px&gt;&lt;/iframe&gt;`</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div id="ojs-cell-14" data-nodetype="expression">

</div>
</div>
</div>
</div>
<div id="tabset-3-4" class="tab-pane" role="tabpanel" aria-labelledby="tabset-3-4-tab">
<div class="cell">
<div class="sourceCode cell-code hidden" id="cb15" data-startfrom="452" data-source-offset="-0"><pre class="sourceCode js code-with-copy"><code class="sourceCode javascript" style="counter-reset: source-line 451;"><span id="cb15-452"><a href="#cb15-452" aria-hidden="true" tabindex="-1"></a><span class="fu">html</span><span class="vs">`&lt;iframe src="./Results/DESeq2/</span><span class="sc">${</span>ora_sel<span class="sc">}</span><span class="vs">/Pathway_Enrichment/Project_</span><span class="sc">${</span>ora_sel<span class="sc">}</span><span class="vs">_kegg_down/Report_</span><span class="sc">${</span>ora_sel<span class="sc">}</span><span class="vs">_kegg_down.html" width:100% height:1200px&gt;&lt;/iframe&gt;`</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div id="ojs-cell-15" data-nodetype="expression">

</div>
</div>
</div>
</div>
</div>
</div>
</section>
<section id="gene-set-enrichment-analysis-1" class="level3 page-columns page-full">
<h3 class="anchored" data-anchor-id="gene-set-enrichment-analysis-1">Gene Set Enrichment Analysis</h3>
<p>Since the knockdown versus control comparisons do not yield a subset of peaks with significantly different coverage, a different enrichment strategy is used for these comparisons. GSEA or gene set enrichment analysis should in theory be suited better to detect enriched biological themes in combinations of genes/peaks that move just a little bit by themselves but in group move in the same direction.</p>
<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Note
</div>
</div>
<div class="callout-body-container callout-body">
<p>To make sense of this long list of peaks, we can perform a gene set enrichment analysis (GSEA). GSEA uses a given statistical metric comparing two conditions to rank all peaks/genes and applies a weighted Kolmogorov–Smirnov (KS) statistic to calculate an Enrichment Score (ES). Basically, all peaks within 2000bp of a Transcription Start Site (1000bp upstream and 1000bp downstream) were collected and ranked according to the log2FoldChange of their respective peak. Next, the ES for different gene sets are calculated running through the entire ranked list increasing the score when a gene in the set is encountered and decreasing the score when the gene encountered is not in the analyzed set. The ES of this set is the maximum difference from 0.</p>
<hr>
<p><strong>Calculation of an Enrichment Score</strong></p>
<ul>
<li>Rank genes by a statistic (e.g.&nbsp;log2FoldChange)</li>
<li>Compute cumulative sum over ranked genes:
<ul>
<li>Increase sum when gene in set, decrease it otherwise.</li>
<li>Magnitude of increment depends on correlation of gene with phenotype.</li>
</ul></li>
<li>Record the maximum deviation from zero as the enrichment score (ES)</li>
</ul>
<hr>
<p>A high (N)ES indicates these genes are found high up in the list. In other words, a high (N)ES value means that for the genes in this set there is - on average - a shift towards more open chromatin. The significance of each (N)ES is calculated by permuting the sets and recomputing the ES, getting a null distribution for the ES for a particular gene set. A multiple comparison correction is also performed on the p values. More info on the GSEA algorithm can be found at [https://bioinformatics.mdanderson.org/MicroarrayCourse/Lectures09/gsea1_bw.pdf].</p>
<p>In this projects we used as a ranking metric the log2FoldChange of the peaks within 2000bp around a TSS. This way genes can be ranked from more open chromatin to mpre closed chromatin. When it comes to the gene sets to be tested, we use the KEGG collection of pathways.</p>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code hidden" id="cb16" data-startfrom="482" data-source-offset="-0"><pre class="sourceCode js code-with-copy"><code class="sourceCode javascript" style="counter-reset: source-line 481;"><span id="cb16-482"><a href="#cb16-482" aria-hidden="true" tabindex="-1"></a>viewof gsea_sel <span class="op">=</span> Inputs<span class="op">.</span><span class="fu">select</span>([<span class="st">"IK_0_vs_C_0"</span><span class="op">,</span> <span class="st">"IK_1_vs_C_1"</span><span class="op">,</span> <span class="st">"IK_4_vs_C_4"</span><span class="op">,</span> <span class="st">"IK_24_vs_C_24"</span><span class="op">,</span> <span class="st">"IK_0_vs_C_0_D12"</span><span class="op">,</span> <span class="st">"IK_0_vs_C_0_D1256"</span>]<span class="op">,</span> { <span class="dt">label</span><span class="op">:</span> <span class="st">"Choose Contrast"</span> })</span>
<span id="cb16-483"><a href="#cb16-483" aria-hidden="true" tabindex="-1"></a><span class="fu">html</span><span class="vs">`&lt;p&gt;&lt;/p&gt;`</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div>
<div id="ojs-cell-16-1" data-nodetype="declaration">

</div>
</div>
</div>
<div class="cell-output cell-output-display">
<div>
<div id="ojs-cell-16-2" data-nodetype="expression">

</div>
</div>
</div>
</div>
<div class="column-body-outset-right">
<div class="cell">
<div class="sourceCode cell-code hidden" id="cb17" data-startfrom="489" data-source-offset="-0"><pre class="sourceCode js code-with-copy"><code class="sourceCode javascript" style="counter-reset: source-line 488;"><span id="cb17-489"><a href="#cb17-489" aria-hidden="true" tabindex="-1"></a><span class="fu">html</span><span class="vs">`&lt;iframe src="./Results/DESeq2/</span><span class="sc">${</span>gsea_sel<span class="sc">}</span><span class="vs">/Pathway_Enrichment/Project_</span><span class="sc">${</span>gsea_sel<span class="sc">}</span><span class="vs">_gsea/Report_</span><span class="sc">${</span>gsea_sel<span class="sc">}</span><span class="vs">_gsea.html" width:100% height:900px&gt;&lt;/iframe&gt;`</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div id="ojs-cell-17" data-nodetype="expression">

</div>
</div>
</div>
</div>
</section>
<section id="comparison-donor-pairs" class="level3">
<h3 class="anchored" data-anchor-id="comparison-donor-pairs">Comparison Donor Pairs</h3>
<p>For the comparison between IK_0 and C_0 we have two sets of donors available. Unfortunately, the overlap between the two is rather low. This is visualzed in figure. Here, the log2(FC) of the two donor sets are plotted against each other. In black, you see the line indicating an identical distribution. In blue you see the regression line indicating almost no congruency.</p>
<p>For comparison, below is an example of more congruency when comparing the IK_1 to C_0 and IK_4 to C_0.</p>
</section>
</section>
</section>
<section id="data-responsibility" class="level1">
<h1>Data Responsibility</h1>
<ul>
<li><strong>NBIS &amp; Uppnex</strong> Unfortunately, we do not have resources to keep any files associated with the support request. We suggest that you safely store the results delivered by us. In addition, we ask that you remove the files from UPPMAX/UPPNEX after analysis is completed. The main storage at UPPNEX is optimized for high-speed and parallel access, which makes it expensive and not the right place for long time archiving.</li>
<li><strong>Sensitive data</strong> Please note that special considerations may apply to the human-derived sensitive personal data. These should be handled according to specific laws and regulations.</li>
<li><strong>Long-term backup</strong> The responsibility for data archiving lies with universities and we recommend asking your local IT for support with long-term data archiving. Also the newly established Data Office at SciLifeLab may be of help to discuss other options.</li>
</ul>
</section>
<section id="acknowledgements" class="level1">
<h1>Acknowledgements</h1>
<p>If you are presenting the results in a paper, at a workshop or conference, we kindly ask you to acknowledge us.</p>
<ul>
<li><p><strong>NBIS Staff</strong> are encouraged to be co-authors when this is merited in accordance to the ethical recommendations for authorship, e.g.&nbsp;<a href="http://www.icmje.org/recommendations/browse/roles-and-responsibilities/defining-the-role-of-authors-and-contributors.html">ICMJE recommendations</a>. If applicable, please include <strong>Vincent van Hoef, National Bioinformatics Infrastructure Sweden, Science for Life Laboratory, Uppsala University</strong>, as co-author. In other cases, NBIS would be grateful if support by us is acknowledged in publications according to this example: <a href="https://www.nbis.se/resources/support.html">“Support by NBIS (National Bioinformatics Infrastructure Sweden) is gratefully acknowledged”</a>.</p></li>
<li><p><strong>UPPMAX</strong> If your project has used HPC resources we kindly asks you to acknowledge UPPMAX and SNIC. If applicable, please add: <a href="https://www.uppmax.uu.se/support/faq/general-miscellaneous-faq/acknowledging-uppmax--snic--and-uppnex/">“The computations were performed on resources provided by SNIC through Uppsala Multidisciplinary Center for Advanced Computational Science (UPPMAX) under Project SNIC XXXX/Y-ZZZ”</a>.</p></li>
<li><p><strong>NGI</strong> In publications based on data from NGI Sweden, the authors must acknowledge SciLifeLab, NGI and UPPMAX: <a href="https://ngisweden.scilifelab.se/info/faq#how-do-i-acknowledge-ngi-in-my-publication">“The authors would like to acknowledge support from Science for Life Laboratory, the National Genomics Infrastructure, NGI, and Uppmax for providing assistance in massive parallel sequencing and computational infrastructure”</a>.</p></li>
</ul>
</section>
<section id="closing-procedures" class="level1">
<h1>Closing Procedures</h1>
<p>You should soon be contacted by one of our managers, <a href="jessica.lindvall@nbis.se">Jessica Lindvall</a> or <a href="henrik.lantz@nbis.se">Henrik Lantz</a>, with a request to close down the project in our internal system and for invoicing matters. If we do not hear from you within <strong>30 days</strong> the project will be automatically closed and invoice sent. Again, we would like to remind you about data responsibility and acknowledgements, see Data Responsibility and Acknowledgments sections.</p>
<p>You are naturally more than welcome to come back to us with further data analysis request at any time via <a href="http://nbis.se/support/support.html">NBIS support</a>.</p>
<p>Thank you for using NBIS and all the best for future research.</p>
</section>

</main>
<!-- /main column -->
<script type="ojs-module-contents">
{"contents":[{"methodName":"interpret","cellName":"ojs-cell-1","inline":false,"source":"viewof sample_sel = Inputs.select(insert_hist, { label: \"Choose Sample\" })\nhtml`<p></p>`\n"},{"methodName":"interpret","cellName":"ojs-cell-2","inline":false,"source":"//| label: fig-insert-hist\n//| fig-cap: \"Insert size histogram.\"\nhtml`<iframe src=\"./Results/bwa/mergedLibrary/picard_metrics/pdf/${sample_sel}.mLb.clN.CollectMultipleMetrics.insert_size_histogram.pdf\" width:100% height:600px></iframe>`\n"},{"methodName":"interpret","cellName":"ojs-cell-3","inline":false,"source":"viewof treat_sel = Inputs.select(contrast, { label: \"Choose Contrast\" })\nhtml`<p></p>`\n"},{"methodName":"interpret","cellName":"ojs-cell-4","inline":false,"source":"md`![Before Normalization](./Results/EDA/MD/pre_norm/${treat_sel}_md.png)`\n"},{"methodName":"interpret","cellName":"ojs-cell-5","inline":false,"source":"md`![After Normalization](./Results/EDA/MD/post_norm/${treat_sel}_md.png)`\n"},{"methodName":"interpret","cellName":"ojs-cell-6","inline":false,"source":"viewof consGC_sel = Inputs.select(consGC, { label: \"Choose Contrast\" })\nhtml`<p></p>`\n"},{"methodName":"interpret","cellName":"ojs-cell-7","inline":false,"source":"md`![Before Normalization](./Results/EDA/GC_Content/pre_norm/${consGC_sel}_LFC_gc_bin.png)`\n"},{"methodName":"interpret","cellName":"ojs-cell-8","inline":false,"source":"md`![After Normalization](./Results/EDA/GC_Content/post_norm/${consGC_sel}_LFC_gc_bin.png)`\n"},{"methodName":"interpret","cellName":"ojs-cell-9","inline":false,"source":"viewof deseq_sel = Inputs.select(deseq, { label: \"Choose Contrast\" })\nhtml`<p></p>`\n"},{"methodName":"interpret","cellName":"ojs-cell-10","inline":false,"source":"md`![Volcano Plot](./Results/DESeq2/${deseq_sel}/${deseq_sel}_volcano.png)`\n"},{"methodName":"interpret","cellName":"ojs-cell-11","inline":false,"source":"viewof ora_sel = Inputs.select([\"C_1_vs_C_0\", \"C_4_vs_C_0\", \"C_24_vs_C_0\"], { label: \"Choose Contrast\" })\nhtml`<p></p>`\n"},{"methodName":"interpret","cellName":"ojs-cell-12","inline":false,"source":"html`<iframe src=\"./Results/DESeq2/${ora_sel}/Pathway_Enrichment/Project_${ora_sel}_go_up/Report_${ora_sel}_go_up.html\" width:100% height:1200px></iframe>`\n"},{"methodName":"interpret","cellName":"ojs-cell-13","inline":false,"source":"html`<iframe src=\"./Results/DESeq2/${ora_sel}/Pathway_Enrichment/Project_${ora_sel}_go_down/Report_${ora_sel}_go_down.html\" width:100% height:1200px></iframe>`\n"},{"methodName":"interpret","cellName":"ojs-cell-14","inline":false,"source":"html`<iframe src=\"./Results/DESeq2/${ora_sel}/Pathway_Enrichment/Project_${ora_sel}_kegg_up/Report_${ora_sel}_kegg_up.html\" width:100% height:1200px></iframe>`\n"},{"methodName":"interpret","cellName":"ojs-cell-15","inline":false,"source":"html`<iframe src=\"./Results/DESeq2/${ora_sel}/Pathway_Enrichment/Project_${ora_sel}_kegg_down/Report_${ora_sel}_kegg_down.html\" width:100% height:1200px></iframe>`\n"},{"methodName":"interpret","cellName":"ojs-cell-16","inline":false,"source":"viewof gsea_sel = Inputs.select([\"IK_0_vs_C_0\", \"IK_1_vs_C_1\", \"IK_4_vs_C_4\", \"IK_24_vs_C_24\", \"IK_0_vs_C_0_D12\", \"IK_0_vs_C_0_D1256\"], { label: \"Choose Contrast\" })\nhtml`<p></p>`\n"},{"methodName":"interpret","cellName":"ojs-cell-17","inline":false,"source":"html`<iframe src=\"./Results/DESeq2/${gsea_sel}/Pathway_Enrichment/Project_${gsea_sel}_gsea/Report_${gsea_sel}_gsea.html\" width:100% height:900px></iframe>`\n"},{"methodName":"interpretQuiet","source":"shinyInput('sample_sel')"},{"methodName":"interpretQuiet","source":"shinyInput('treat_sel')"},{"methodName":"interpretQuiet","source":"shinyInput('consGC_sel')"},{"methodName":"interpretQuiet","source":"shinyInput('deseq_sel')"},{"methodName":"interpretQuiet","source":"shinyInput('ora_sel')"},{"methodName":"interpretQuiet","source":"shinyInput('gsea_sel')"}]}
</script>
<script type="module">
window._ojs.paths.runtimeToDoc = "../../..";
window._ojs.paths.runtimeToRoot = "../../..";
window._ojs.paths.docToRoot = "";
window._ojs.selfContained = false;
window._ojs.runtime.interpretFromScriptTags();
</script>
<script type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    setTimeout(function() {
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      let href = ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const cites = ref.parentNode.getAttribute('data-cites').split(' ');
    tippyHover(ref, function() {
      var popup = window.document.createElement('div');
      cites.forEach(function(cite) {
        var citeDiv = window.document.createElement('div');
        citeDiv.classList.add('hanging-indent');
        citeDiv.classList.add('csl-entry');
        var biblioDiv = window.document.getElementById('ref-' + cite);
        if (biblioDiv) {
          citeDiv.innerHTML = biblioDiv.innerHTML;
        }
        popup.appendChild(citeDiv);
      });
      return popup.innerHTML;
    });
  }
});
</script>
</div> <!-- /content -->



</body></html>